geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/TiledRasterRDD.scala

```diff
@@ -211,6 +211,53 @@ abstract class TiledRasterRDD[K: SpatialComponent: AvroRecordCodec: JsonFormat:
     withRDD(rdd.convert(CellType.fromName(newType)))
 
   protected def withRDD(result: RDD[(K, MultibandTile)]): TiledRasterRDD[_]
+
+  def singleTileLayerRDD: TileLayerRDD[K] = TileLayerRDD(
+    rdd.map({ case (k, v) => (k, v.band(0)) }),
+    rdd.metadata
+  )
```

Comment:
Having a think about it, I think this is good. I imagine future function will be overloaded on the type. I expect that in multiband case you would not always apply the same summary for all the bands. ex:
```py
stats = rdd.polygonalSummary(geom, [MAX, MAX, AVERAGE])
# stats is [float, float, float]
```